### PR TITLE
Request to iamoracle as a delegate

### PR DESCRIPTION
**Pull request by iamoracle**

### CODE REVIEW

File: src/config/delegates.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 {
+  "0xC20F4412fa84b33e0f3657113736F229a811A285": {
+    "name": "iamoracle",
+    "address": "0xC20F4412fa84b33e0f3657113736F229a811A285",
+    "logoUri": "/logos/delegatees/iamoracle.jpeg",
```
Comment (aaronmgdr) [inline src/config/delegates.json:5]: file is png but this is jpeg

**Review by aaronmgdr — CHANGES_REQUESTED** [review]

**Review by iamoracle — COMMENTED** [review]
Fixed

**Comment by aaronmgdr** [comment]
hmm when i visit preview your logo does not show. im not sure why.

**Comment by iamoracle** [comment]
Fixed now @aaronmgdr

**Review by jmrossy — APPROVED** [review]

**Pull request merged**